source/pgg_mode/PGGWorldSetup.cc

```diff
@@ -4,61 +4,58 @@
 #include "PGGWorld.h"
 #include "PGGHost.h"
 #include "PGGSymbiont.h"
+
 /**
- * Input: The number of PGG hosts.
+ * Input: None.
  *
- * Output: None.
+ * Output: A pointer to a new PGG host.
  *
- * Purpose: To populate the world with PGG hosts with appropriate phenotypes.
+ * Purpose: To create and get the pointer to a new PGG host.
  */
-void PGGWorld::SetupHosts(long unsigned int* POP_SIZE){
-  for (size_t i = 0; i < *POP_SIZE; i++) {
-    emp::Ptr<PGGHost> new_org;
-    new_org.New(&GetRandom(), this, my_config, my_config->HOST_INT());
-    InjectHost(new_org);
-  }
+emp::Ptr<Organism> PGGWorld::GetNewHost() {
+  emp::Ptr<PGGHost> new_org;
+  new_org.New(&GetRandom(), this, my_config, my_config->HOST_INT());
```

Comment:
All of the config variable names will need to be updated once the config inheritance PR is pulled in